packages/components/src/Workflow/subcomponents/Wrapper/index.tsx

```diff
@@ -0,0 +1 @@
+export * from "./Wrapper"
```

Comment:
Thoughts on renaming these `index.tsx` files to `index.ts` while you're here? 😄

Reply:
Could hurt to tackle that in one fell swoop

Reply:
💔

Reply:
** couldn't 😆 Should be good for another review now